### ISSUE TITLE
error: unrecognized arguments: --model_dir

### ISSUE THREAD

**Issue by Pooja199** [aws/sagemaker-python-sdk#1720]
**Describe the bug**
The script is not picking the model_dir given

**To reproduce**
sagemaker_session = sagemaker.Session()

role = get_execution_role()

output_path = 's3://sagemaker-ml-circuit-breaker/lstm-data/models/'

estimator = TensorFlow(entry_point='train.py',
                           source_dir='src',
                           role=role,
                           train_instance_count=1,
                           train_instance_type='ml.m5.large',
                           py_version='py3',
                           framework_version='2.0.1',
                           output_path = output_path)
estimator.fit({'train': 's3://sagemaker-ml-circuit-breaker/lstm-data/train/', 
               'test': 's3://sagemaker-ml-circuit-breaker/lstm-data/test/'})

In train.py: 
from detector import Detector
from datetime import datetime as dt
import argparse
import os

def _parse_args():
    parser = argparse.ArgumentParser(description='Parse path to anomaly labels if provided.')
    parser.add_argument('-l', '--labels_path', default=None, required=False)

    parser.add_argument('--train', default=os.environ['SM_CHANNEL_TRAIN'])
    parser.add_argument('--test', default=os.environ['SM_CHANNEL_TEST'])
    parser.add_argument('--model_dir', type=str, default='')

    
    args = parser.parse_args()
    return args

if __name__ == '__main__':
    args = _parse_args()
    print("input data " + os.listdir(args.train))
    detector = Detector(labels_path=args.labels_path)
    detector.input_data = args.input_train
    detector.run()

**Expected behavior**
It should be able to run the script file without any unrecognized argument error

**Screenshots or logs**
logs: 
Invoking script with the following command:

/usr/bin/python3 -m train --model_dir s3://sagemaker-ml-circuit-breaker/lstm-data/models/tensorflow-training-2020-07-16-22-19-36-006/model

Using TensorFlow backend.
usage: train.py [-h] [-l LABELS_PATH] [--train TRAIN] [--test TEST]
                [--sm_model_dir SM_MODEL_DIR]
train.py: error: unrecognized arguments: --model_dir s3://sagemaker-ml-circuit-breaker/lstm-data/models/tensorflow-training-2020-07-16-22-19-36-006/model
2020-07-16 22:22:21,331 sagemaker-containers ERROR    ExecuteUserScriptError:
Command "/usr/bin/python3 -m train --model_dir s3://sagemaker-ml-circuit-breaker/lstm-data/models/tensorflow-training-2020-07-16-22-19-36-006/model"

**System information**
A description of your system. Please provide:
- **SageMaker Python SDK version**: 
- **Framework name (eg. PyTorch) or algorithm (eg. KMeans)**: Tensorflow 
- **Framework version**: 2.0.1
- **Python version**: Python 3.6.10
- **CPU or GPU**: CPU
- **Custom Docker image (Y/N)**: N

**Additional context**

**Comment by Pooja199** [comment]
ignore this .. closing this as i was able to resolve the issue
